plugins/keepconf/keepconf_status

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+keepconf_report -Indicate the status of keepconf execution
+
+=head1 APPLICABLE SYSTEMS
+Linux box with keepconf installed.
+
+=head1 CONFIGURATION
+
+[keepconf_*]
+env.files /var/tmp/keepconf-mon.csv
+# Multifile its available
+#env.files /var/tmp/keepconf-client1.csv /var/tmp/keepconf-client2.csv
+
+=head1 AUTHOR
+
+Ricardo F. <rfrail3@yahoo.es>
+
+=head1 VERSION
+v.1.2.0
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGICK MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+. $MUNIN_LIBDIR/plugins/plugin.sh
+
+LIST=$files
+
+if [ "$1" = "autoconf" ]; then
+
+	for j in ${LIST}; do
+		if [ ! -f ${j} ]; then
+			echo no
+			exit 1
+		fi
+	done
+
+	echo yes
+	exit 0
+fi
+
+if [ "$1" = "config" ]; then
+
+	echo 'graph_title Keepconf report'
+	echo 'graph_args --base 1000 -l 0 '
+	echo 'graph_vlabel Hosts'
+	echo 'graph_scale no'
+	echo 'graph_category keepconf'
```

Comment:
please pick one of the [commonly used categories](http://munin-monitoring.org/wiki/graph_category_list), if one of them matches somehow